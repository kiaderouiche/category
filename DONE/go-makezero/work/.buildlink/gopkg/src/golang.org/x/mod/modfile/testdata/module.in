/usr/pkg/gopkg/src/golang.org/x/mod/modfile/testdata/module.in