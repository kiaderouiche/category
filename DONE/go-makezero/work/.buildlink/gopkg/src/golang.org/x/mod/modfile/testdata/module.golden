/usr/pkg/gopkg/src/golang.org/x/mod/modfile/testdata/module.golden